client/src/logger.rs

```diff
@@ -63,6 +63,38 @@ pub fn log(level: LevelFilter) {
         })
         .chain(fern::log_file(&fs::log_file()).expect("Failed to setup log file!"));
 
+    #[allow(unused_mut)]
+    let mut color = colored::control::SHOULD_COLORIZE.should_colorize();
+
+    // Safety:
+    // GetStdHandle is checked for errors and for null handles.
+    // GetConsoleMode the handle passed must have the GENERIC_READ access right
+    // this always the case for handles returned by
+    // GetStdHandle(STD_OUTPUT_HANDLE) unless the program has changed the access
+    // rights, we don't so it's safe. The second argument lpMode must be a valid
+    // pointer for the duration of the call guaranteed by the rust borrow
+    // checker since mode lives until the end of the block.
+    #[cfg(windows)]
+    unsafe {
```

Comment:
May this be moved over to `windows.rs`? It encapsulates all windows related things and this would fit there well.